packages/@webex/widgets/package.json

```diff
@@ -0,0 +1,156 @@
+{
+  "name": "@webex/widgets",
+  "version": "1.27.4",
+  "engines": {
+    "node": ">=20.13.1"
+  },
+  "main": "dist/webexWidgets.esm.js",
+  "module": "dist/webexWidgets.esm.js",
+  "scripts": {
+    "build": "rollup -c",
+    "demo:serve": "webpack serve ./demo/App.jsx --mode development",
+    "demo:build": "webpack ./demo/App.jsx --mode production",
+    "release": "semantic-release",
+    "release:debug": "semantic-release --debug",
+    "release:dry-run": "semantic-release --dry-run",
+    "start": "npm run demo:serve",
+    "test:e2e": "npm run demo:build && wdio wdio.conf.js",
+    "test:eslint": "eslint src/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webex/widgets.git"
+  },
+  "publishConfig": {
+    "tag": "latest",
```

Comment:
Are we planning to publish this onto the same tag in npm?

Reply:
In this PR, this is correct so that if we do any bug fixes mid-way, we can publish it to the latest. We will change it in the feature branch later.